src/tools/dev/scripts/bv_support/bv_vtk.sh

```diff
@@ -961,6 +960,15 @@ function build_vtk
         vopts="${vopts} -Dembree_DIR=${EMBREE_INSTALL_DIR}"
     fi
 
+    # zlib support, use the one we build
+    vopts="${vopts} -DVTK_USE_SYSTEM_ZLIB:BOOL=ON"
```

Comment:
Just wanna confirm VTK's `-DVTK_USE_SYSTEM_ZLIB:BOOL` is a switch that is used to make VTK use a ZLIB of our choosing instead of its own?

Reply:
Correct.